File: ExampleMvvm/librarycommon/src/main/java/com/degde/library/library/login/model/ModelLoginResponse.java
package com.degde.library.library.login.model;


import androidx.lifecycle.ViewModel;

import com.google.gson.annotations.SerializedName;

public class ModelLoginResponse extends ViewModel {

    @SerializedName("access_token")
    private String access_token;
    @SerializedName("expires_in")
    private int expires_in;
    @SerializedName("id_token")
    private String id_token;
    @SerializedName("refresh_token")
    private String refresh_token;
    @SerializedName("token_type")
    private String token_type;
    @SerializedName("expires_at")
    private String expires_at;

    public String getAccess_token() {
        return access_token;
    }

    public void setAccess_token(String access_token) {
        this.access_token = access_token;
    }

    public int getExpires_in() {
        return expires_in;
    }

    public void setExpires_in(int expires_in) {
        this.expires_in = expires_in;
    }

    public String getId_token() {
        return id_token;
    }

    public void setId_token(String id_token) {
        this.id_token = id_token;
    }

    public String getRefresh_token() {
        return refresh_token;
    }

    public void setRefresh_token(String refresh_token) {
        this.refresh_token = refresh_token;
    }

    public String getToken_type() {
        return token_type;
    }

    public void setToken_type(String token_type) {
        this.token_type = token_type;
    }

    public String getExpires_at() {
        return expires_at;
    }

    public void setExpires_at(String expires_at) {
        this.expires_at = expires_at;
    }
}

File: ExampleMvvm/corekotlin/src/main/java/com/core/mvvm/login/model/ModelLoginResponse.kt
package com.core.mvvm.login.model

import com.google.gson.annotations.SerializedName

class ModelLoginResponse {
    @SerializedName("access_token")
    private var access_token: String? = null
    @SerializedName("expires_in")
    private var expires_in = 0
    @SerializedName("id_token")
    private var id_token: String? = null
    @SerializedName("refresh_token")
    private var refresh_token: String? = null
    @SerializedName("token_type")
    private var token_type: String? = null
    @SerializedName("expires_at")
    private var expires_at: String? = null

    fun getAccess_token(): String? {
        return access_token
    }

    fun setAccess_token(access_token: String?) {
        this.access_token = access_token
    }

    fun getExpires_in(): Int {
        return expires_in
    }

    fun setExpires_in(expires_in: Int) {
        this.expires_in = expires_in
    }

    fun getId_token(): String? {
        return id_token
    }

    fun setId_token(id_token: String?) {
        this.id_token = id_token
    }

    fun getRefresh_token(): String? {
        return refresh_token
    }

    fun setRefresh_token(refresh_token: String?) {
        this.refresh_token = refresh_token
    }

    fun getToken_type(): String? {
        return token_type
    }

    fun setToken_type(token_type: String?) {
        this.token_type = token_type
    }

    fun getExpires_at(): String? {
        return expires_at
    }

    fun setExpires_at(expires_at: String?) {
        this.expires_at = expires_at
    }
}
